docs/lsm/partial_compiles/partial.cf

```diff
@@ -0,0 +1,79 @@
+import lsm
+import lsm::fsm
+import std::testing
+
+entity Router extends lsm::ServiceEntity:
+    """
+        A service for managing routers
+    """
+    string mgmt_ip
+end
+
+index Router(instance_id)
+
+entity Port extends lsm::ServiceEntity:
+    """
+        A service for managing ports on routers
+    """
+    string name
+end
+
+index Port(instance_id)
+
+Port.router [1] lsm::__service__, lsm::__rwplus__ Router
+""" An Inter Service Relation between Router and Port"""
+
+implementation router_config for Router:
+    """ Add a dummy resource to the router to represent actual configuration """
+    self.resources += std::testing::NullResource(name=self.mgmt_ip)
```

Comment:
This doesn't exist, does it? Which would be a problem if we use this snippet for testing.